service_group.go

```diff
@@ -142,7 +142,7 @@ func (s *service) activateGroup(ctx context.Context, pk crypto.PubKey, localOnly
 	dbOpts := &iface.CreateDBOptions{LocalOnly: &localOnly}
 	gc, err := s.odb.OpenGroup(ctx, g, dbOpts)
 	if err != nil {
-		return errcode.ErrGroupOpen.Wrap(err)
+		return err
```

Comment:
why did you remove this errcode? Should you also remove this errcode in the errcode.proto file?